src/openrct2/drawing/Drawing.h

```diff
@@ -296,6 +296,33 @@ enum class FilterPaletteID : int32_t
     PaletteGlassDarkPink = PaletteGlass + COLOUR_DARK_PINK,
     PaletteGlassBrightPink = PaletteGlass + COLOUR_BRIGHT_PINK,
     PaletteGlassLightPink = PaletteGlass + COLOUR_LIGHT_PINK,
+    PaletteG2 = 144,
+    PaletteG2Invisible = 166,
+    PaletteGlassG2 = 168,
+    PaletteGlassDarkOliveDark = PaletteGlassG2,
```

Comment:
This should all be in order from L298 so that GlassDarkOliveDark is just PaletteGlass + COLOUR_DARK_OLIVE_DARK